.eslintrc

```diff
@@ -1,40 +1,72 @@
 {
```

Comment:
We should consider getting parity across eslintrc files in eq products

Reply:
They have different rules enabled where necessary, was there anything specific you meant?. Main difference I see is validator we use single quote, but we chose double a while ago, so that should change other repos.

Reply:
yeah I should have been clearer - where there are settings that are required across repos we should keep them consistent. The string quotations are a really good example of where we should be using the same settings